container_images/test-manager/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM golang
+
+WORKDIR /go/src/
+COPY ./ ./guest-test-infra
```

Comment:
does this work? if you're using legacy GOPATH builds doesn't this need to be /go/src/github.com/GoogleCloudPlatform/guest-test-infra ?